wafer/talks/templates/wafer.talks/talk.html

```diff
@@ -41,6 +41,13 @@
 <div class="well">
     {{ object.abstract.rendered|safe }}
 </div>
+{% if show_private_notes and perms.edit_private_notes and object.private_notes %}
```

Comment:
I like having this information here so all everything related to the talk is immediately accessible for reviewers. If we're not going to show the notes here, we will need to document how people reviewing and commenting on the talks get access to this information - whether via the admin interface or via some other form.